File: java/program/Storage.java
class Storage {
    protected String capacity, type;
    public Storage(String capacity, String type) {
        this.capacity = capacity;
        this.type = type;
    }

    public String getCapacity() { return capacity; }
    public String getType() { return type; }

    // setter
    public void setCapacity(String capacity) { this.capacity = capacity; }
    public void setType(String type) { this.type = type; }
    

}

File: cpp/program/Storage.cpp
#pragma once

#include <iostream>
#include <vector>
using namespace std;

class Storage {
protected:
    int capacity;     // Kapasitas (GB)
    string type;      // Jenis storage 

public:
    Storage(int cap, string t) : capacity(cap), type(t) {}

    // Getter
    int getCapacity() const { return capacity; }  
    string getType() const { return type; }      

    // Setter
    void setCapacity(int cap) { capacity = cap; }
    void setType(string t) { type = t; }
};